config.proto

```diff
@@ -291,7 +291,7 @@ message Config {
     /*
      * acquire an address via DHCP or assign static
      */
-    EthMode wifi_mode = 2;
+    EthMode eth_mode = 2;
 
```

Comment:
err - no - this one is for fixed ip on wifi. we now have 2 settings that are named the same.

Reply:
Can we rename EthMode then? Maybe ProtocolMode?